src/nyc_trees/apps/home/routes.py

```diff
@@ -19,3 +21,7 @@
 
 training_list_page = route(GET=do(render_template('home/training.html'),
                                   v.training_list_page))
+
+groups_to_follow = route(GET=do(render_template('home/groups_to_follow.html'),
+                                update({'chunk_size': 2}),
```

Comment:
Consider moving `chunk_size` to the view. They contain data needed to render the templates so I think that's where most people would expect to find this.